### ISSUE TITLE
You don't seem to have any documentation on how to use this with the current version of Angular

### ISSUE THREAD

**Issue by Neutrino-Sunset** [AzureAD/microsoft-authentication-library-for-js#7405]
### Wrapper Library

MSAL Angular (@azure/msal-angular)

### Public or Confidential Client?

Public

### Documentation Location

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/initialization.md

### Description

Your initialization documentation [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/initialization.md) describes updating `NgModule` which is no longer part of the default Angular application.

Your upgrade steps [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-v3-upgrade-guide.md) don't describe the actual steps required to upgrade an application.

And you have an Angular 3 sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v3-samples/angular18-standalone-sample) where the readme file does not describe any of the steps required to add MSAL to the base application. The first step instructs the modification of a file that isn't even part of a base application (editing `app.config.ts`).

In short, it appears that you have no up-to-date documentation whatsoever for MSAL Angular v3 and current Angular.

Do you really think this library is useable with the documentation in this kind of state?

**Comment by Neutrino-Sunset** [comment]
This is the initialization page [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/initialization.md). First one has to realize that for any of the documentation to be useful at all one has to modify a standard new Angular application to use a root module.

After that you will hit the issue that the documentation misses a key element of initialization which is the need to add `<app-redirect></app-redirect>` to `index.html`.

After wading through the reams of imports and configuration require to initialize MSAL, the next thing a user is likely to want to do is just sign a user in.

However the documentation does not seem to describe this process anywhere. Instead it starts going on about guards and interceptors. This is not helpful. Technical documenation should first describe the simple scenarios, and only then go on to more complex ones. The user may not even be using routing.

At the bottom of the page it says

> You are ready to use @azure/msal-angular [public APIs](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/public-apis.md)!

The 'Public APIs for MSAL Angular' [page](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/public-apis.md) that this links to, is also linked from the 'Usage' section of the main documentation index.

This page states that

> The login and acquire token functions using Angular observables are found on the [IMsalService](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/src/IMsalService.ts/).

However there does not appear to be any documentation whatsoever describing how to actually use this api. All links just go directly to the raw souce.

In the entire `Usage` section of the documentation [index](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/README.md#installation) I can find no information anywhere on how to log a user in or log a user out.
